frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-utilization.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import {
+  Dropdown,
+  humanizeDecimalBytes,
+  humanizeCpuCores as humanizeCpuCoresUtil,
+} from '@console/internal/components/utils';
+import { getName, getNamespace } from '@console/shared';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import UtilizationBody from '@console/shared/src/components/dashboard/utilization-card/UtilizationBody';
+import UtilizationItem from '@console/shared/src/components/dashboard/utilization-card/UtilizationItem';
+import {
+  ONE_HR,
+  SIX_HR,
+  TWENTY_FOUR_HR,
+  UTILIZATION_QUERY_HOUR_MAP,
+} from '@console/shared/src/components/dashboard/utilization-card/dropdown-value';
+import { getRangeVectorStats } from '@console/internal/components/graphs/utils';
+import { VMDashboardContext } from '../../vms/vm-dashboard-context';
+import { findVMPod } from '../../../selectors/pod/selectors';
+import { getUtilizationQueries, VMQueries } from './queries';
+
+const metricDurations = [ONE_HR, SIX_HR, TWENTY_FOUR_HR];
+const metricDurationsOptions = _.zipObject(metricDurations, metricDurations);
+
+const readPrometheusResult = (prometheusResults, query) => {
+  const data = prometheusResults.getIn([query, 'data']) as PrometheusResponse;
+  const error = prometheusResults.getIn([query, 'loadError']);
+  return [data, error];
+};
+
+// TODO: extend humanizeCpuCores() from @console/internal for the flexibility of space
+const humanizeCpuCores = (v) => {
```

Comment:
why adding space between ? We should stay consistent with the rest of the console

Reply:
Other values are with space between value and unit. @matthewcarleton , @andybraren , can you advise, please? The question is about millicores for CPU usage.

Reply:
I suggest to go on with the discussion, but its outcome can be implemented in a follow-up as the development freeze is here and we are in different time-zones.